tensorflow/core/kernels/data/name_utils.cc

```diff
@@ -41,15 +41,23 @@ string OpName(const string& dataset_type) {
     return "PrefetchDataset";
   } else if (dataset_type == "Range") {
     return "RangeDataset";
+  } else if (dataset_type == "Shard") {
```

Comment:
it seems like all of these are `dataset_type + "Dataset"`, would it make sense to simplify the code that way (making "Dataset" a named constant used both here and in `DatasetDebugString`)?

Reply:
Yeah, good suggestion! Changed them.